### PR TITLE
doc: document virtualization setup (passthrough & sriov)

### PR DESCRIPTION
**Pull request by dvrogozh**
Signed-off-by: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -110,6 +110,23 @@ to interact with the service(s) running inside the container:
...
+--------------
+
+Intel GPUs support 2 types of Virtualization setup:
+
+* GPU Passthrough Virtualization
+
+* GPU SR-IOV Virtualization
+
+The first one, GPU Passthrough Virtualization, is a legacy Virtualization
+Technology which allows exclusive access to GPU from under Virtual Machine (VM).
```
Comment (dsocek) [inline README.rst:123]: I'd drop "from"

File: doc/virtualization.rst [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+--------
+
+This guide describes 2 types of GPU Virtualization setup:
+
+* GPU Passthrough Virtualization
+
+* GPU SR-IOV Virtualization
+
+The first one, GPU Passthrough Virtualization, is a legacy Virtualization
+Technology which allows exclusive access to GPU from under Virtual Machine (VM).
```
Comment (dsocek) [inline doc/virtualization.rst:16]: drop "from"

File: doc/virtualization.rst [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+
+   &#x2713;
+
+.. |cross| raw:: html
+
+   &#x2717;
+
+In this article we will provide VM setup instructions assuming the following:
+
+* Ubuntu 20.04 is the Operating System being installed both for the host and VM
```
Comment (dsocek) [inline doc/virtualization.rst:44]: "for" = "on"

File: doc/virtualization.rst [hunk 4]
```diff
@@ -0,0 +1,461 @@
...
+    Advanced -> PCI Configuration -> Memory Mapped I/O above 4GB : Enabled
+    Advanced -> PCI Configuration -> MMIO High Base : 56T
+    Advanced -> PCI Configuration -> Memory Mapped I/O Size : 1024G
+    Advanced -> System Acoustic and Performance Configuration -> Set Fan Profile: Performance
+
+GPU Passthough Virtualization
+-----------------------------
+
+Virtual Machine (VM) setup with GPU passthrough is a type of setup which
+allows exclusive access to GPU from under VM. Mind that with this setup
```
Comment (dsocek) [inline doc/virtualization.rst:68]: drop from

File: doc/virtualization.rst [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+only one VM to which GPU was explicitly assigned will be able to use GPU.
+Other VMs and even a host won't have full access to the GPU.
```
Comment (dsocek) [inline doc/virtualization.rst:70]: better to spell out "won't" into "would not"

File: doc/virtualization.rst [hunk 6]
```diff
@@ -0,0 +1,461 @@
...
+    VM_IMAGE=/opt/vmimage/ubuntu-hdd.qcow2
+    HOST_IP=$(hostname -I | cut -f1 -d ' ')
+    VNC_PORT=40
+    qemu-system-x86_64 -enable-kvm -drive file=$VM_IMAGE \
+      -cpu host -smp cores=8 -m 64G -serial mon:stdio \
+      -device vfio-pci,host=4d:00.0 \
+      -net nic -net user,hostfwd=tcp::10022-:22,hostfwd=tcp::8080-:8080 \
+      -vnc $HOST_IP:$VNC_PORT
+
+At this point you should have running VM with attached GPU in passthrough mode.
```
Comment (dsocek) [inline doc/virtualization.rst:189]: a running VM with an attached..

**Review by dsocek — APPROVED** [review]
Some text related changes suggested.  Otherwise looks good to me

File: README.rst (hunk 1 above)
Comment (dvrogozh) [reply README.rst:123]: done

File: doc/virtualization.rst (hunk 2 above)
Comment (dvrogozh) [reply doc/virtualization.rst]: done

File: doc/virtualization.rst (hunk 3 above)
Comment (dvrogozh) [reply doc/virtualization.rst:45]: done

File: doc/virtualization.rst (hunk 4 above)
Comment (dvrogozh) [reply doc/virtualization.rst]: done

File: doc/virtualization.rst (hunk 5 above)
Comment (dvrogozh) [reply doc/virtualization.rst]: done, but spelled "will not"

File: doc/virtualization.rst (hunk 6 above)
Comment (dvrogozh) [reply doc/virtualization.rst]: done

**Pull request merged**